lib/ecstatic.js

```diff
@@ -57,27 +57,39 @@ var ecstatic = module.exports = function (dir, options) {
     catch (err) {
       return status[400](res, next, { error: err });
     }
-
-    var file = path.normalize(
-          path.join(root,
-            path.relative(
-              path.join('/', baseDir),
-              pathname
+    
+    // try and serve assets
+    var assetsDirectory = '_ecstatic-assets'
```

Comment:
I will change this directory to something like `/_icons`, and make it so that it won't get served up unless the option is on.  I'll embed the non-icon page styles directly in a `<style>` tag since there are only 2 rules.